internal/namespaces/baremetal/v1alpha1/custom_server.go

```diff
@@ -0,0 +1,95 @@
+package baremetal
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	baremetal "github.com/scaleway/scaleway-sdk-go/api/baremetal/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	serverActionTimeout = 20 * time.Minute
+)
+
+func serverWaitCommand() *core.Command {
+	type serverWaitRequest struct {
+		ServerID string
+		Zone     scw.Zone
+	}
+
+	return &core.Command{
+		Short:     `Wait for a server to reach a stable state`,
+		Long:      `Wait for a server to reach a stable state. This is similar to using --wait flag on other action commands, but without requiring a new action on the server.`,
+		Namespace: "baremetal",
+		Resource:  "server",
+		Verb:      "wait",
+		ArgsType:  reflect.TypeOf(serverWaitRequest{}),
```

Comment:
not sure why we wouldn't use `*baremetal.WaitForServerRequest` here, timeout is an interesting argument to the user as well, and we could set its default value (`20s`), which will be visible from the doc

Reply:
If we do this we shlould also update instante Wait server

Reply:
Can be done in a separate PR